examples/configurator-values.yaml

```diff
@@ -0,0 +1,545 @@
+# -- Override the name of the chart
+nameOverride: ""
+# -- Override the full name of the release
+fullnameOverride: ""
+
+# -- Name of the Kubernetes cluster monitored. Can be configured also with `global.cluster`.
+# Note it will be set as an external label in prometheus configuration, it will have precedence over `config.common.external_labels.cluster_name`
+# and `customAttributes.cluster_name``.
+cluster: "test-cluster"
```

Comment:
I would replace this with <FILL IN CLUSTER NAME> so it forces them to do it since that's not valid yaml